frontend/src/safety.ts

```diff
@@ -163,6 +164,18 @@ export function getSafetyRating(
       icon: HiOutlineVideoCamera,
       showOnSummaryOrDetails: "both",
     })
+  } else if (
+    summary.metadata.permissions.devices?.some(
+      (x) => x.toLowerCase() === "input",
```

Comment:
there is no `--device=input` for flatpak. Not sure what this is about.

Reply:
I swore I had seen it in the wild before, but yeah it doesn't exist.

Reply:
I don't think it has reached any app yet - might still take some time for compatibility, but it does exist? https://github.com/flatpak/flatpak/pull/5481

Reply:
Ah it is very new, nevermind.